command/src/main/java/com/jagrosh/jdautilities/command/Command.java

```diff
@@ -35,7 +35,7 @@
  * a low level of development. 
  * <br>All Commands extending this class can define any number of these fields in a object constructor and then 
  * create the command action/response in the abstract 
- * {@link com.jagrosh.jdautilities.command.Command#execute(CommandEvent) #execute(CommandEvent)} body:
+ * {@link Command#execute(CommandEvent) #execute(CommandEvent)} body:
```

Comment:
This breaks JDA Documentation standards, the original way this was done should have been maintained